stdlib/2and3/grp.pyi

```diff
@@ -0,0 +1,10 @@
+from typing import List, NamedTuple
+
+struct_group = NamedTuple("struct_group", [("gr_name", str),
+                                           ("gr_passwd", str),
+                                           ("gr_gid", int),
+                                           ("gr_mem", List[str])])
```

Comment:
> so more constrained types are the right thing to do.

I agree.

> So make gr_passwd an Optional[str], and then we can merge this?

Yes, I just updated type for `gr_passwd`. Please check & merge this.